File: PRAKTIKUM MODUL 6/PRAK605-2410817310009-Muhammad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main() {
    int Matriks_A[10][10], Matriks_B[10][10], Matriks_AXB[10][10];
    int m, g, z, i, jumlah = 0;
    
    scanf("%d", &i);
    printf("Matriks A\n");
    for(m = 0; m < i; m++) {
        for(g = 0; g < i; g++) {
            scanf("%d", &Matriks_A[m][g]);
        }
    }
    printf("Matriks B\n");
    for(m = 0; m < i; m++) {
        for(g = 0; g < i; g++) {
            scanf("%d", &Matriks_B[m][g]);
        }
    }
    for(m = 0; m < i; m++) {
        for(g = 0; g < i; g++) {
            for(z = 0; z < i; z++) {
                jumlah = jumlah + Matriks_A[m][z] * Matriks_B[z][g];
            }
            Matriks_AXB[m][g] = jumlah;
            jumlah = 0;
        }
    }
    printf("Matriks AXB\n");
    for(m = 0; m < i; m++) {
        for(g = 0; g < i; g++) {
            printf("%d ", Matriks_AXB[m][g]);
        }
        printf("\n");
    }
}

File: PRAKTIKUM MODUL 6/PRAK605-2410817310009-Muhammad_Ghazi_Rakhmadi.py
r = int(input())

MatriksA = []
print('Matriks A')
for i in range (r):
    k = list(map(int, input().split()))
    MatriksA.append(k)

MatriksB = []
print('Matriks B')
for i in range (r):
    k = list(map(int, input().split()))
    MatriksB.append(k)
    
MatriksAXB = []
print('Matriks AXB')
for i in range (r):
    MatriksAXB.append([])
    for j in range (r):
        x = 0
        for k in range (r):
            x = x + MatriksA[i][k] * MatriksB[k][j]
        MatriksAXB[i].append(x)
for baris in MatriksAXB:
    for k in baris:
        print(k, end=' ')
    print()